great-idea/index.html

```diff
@@ -16,8 +16,112 @@
 
   <body>
 
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
+<section class="container">
```

Comment:
```suggestion
<div class="container">
```
You should have a div that wraps ALL of the content in your html page.